tuning/config/peft_config.py

```diff
@@ -46,7 +46,7 @@ class LoraConfig:
     r: int = 8
     lora_alpha: int = 32
     target_modules: List[str] = field(
-        default_factory=lambda: ["q_proj", "v_proj"],
+        default=None,
```

Comment:
It would be good to have a unit test that uses the llama model, this LoRA config, does LoRA tuning and gets apater config and ensures adapter config still have q_proj , v_proj